db/system_keyspace.cc

```diff
@@ -1464,19 +1464,17 @@ struct local_cache {
 };
 
 future<std::unordered_map<gms::inet_address, locator::endpoint_dc_rack>> system_keyspace::load_dc_rack_info() {
-    auto msg = co_await execute_cql(format("SELECT peer, data_center, rack from system.{}", PEERS));
+    const auto msg = co_await execute_cql(format("SELECT peer, data_center, rack from system.{}", PEERS));
 
     std::unordered_map<gms::inet_address, locator::endpoint_dc_rack> ret;
     for (const auto& row : *msg) {
-        net::inet_address peer = row.template get_as<net::inet_address>("peer");
         if (!row.has("data_center") || !row.has("rack")) {
             continue;
         }
-        gms::inet_address gms_addr(std::move(peer));
-        sstring dc = row.template get_as<sstring>("data_center");
-        sstring rack = row.template get_as<sstring>("rack");
```

Comment:
Some people would argue that introducing such variables makes the code more readable. I don't have a strong opinion, but it's preferred to avoid such subjective stylistic changes. (I won't oppose this change.)

(The coroutinization below is definitely desired though.)